logback-classic/src/test/java/ch/qos/logback/classic/pattern/ThrowableProxyConverterTest.java

```diff
@@ -206,18 +206,95 @@ public void shouldLimitTotalLinesExcludingSkipped() throws Exception {
         String[] lines = result.split(CoreConstants.LINE_SEPARATOR);
         assertThat(lines).hasSize(3 + 1);
     }
+    
+    @Test
+    public void nullMessage() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2370881413/issues/source?bid=3222618&fileBranchId=3222653#l211)